### DISCUSSION TITLE
HELP PLS

### DISCUSSION THREAD

**Discussion by Dennis120193** [iotaledger/firefly#2327] [Help & Support] [answered]
Hello, I unfortunately premated my pc and deleted my 24-word PIN, so I can no longer access my acc, what can I do? i have the bitpanda transaction on my firefly acc as proof that it's mine i have almost 600 ioatas on it... how do i get the acc back? please urgently help i'm desperate.

**Comment by MiKeF05** [comment, marked as answer]
Unfortunatly you only have the ability to restore your wallet with your 24 words or a stronghold backup + password. Have you done such a backup of your account?
